### DISCUSSION TITLE
Help to understand syntax of templater

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [SilentVoid13/Templater#1135] [Help]
* What is tR? What does it do? I have seen it in some scripts, but I couldn't understand.
* How do I add templater syntax in YAML frontmatter, it gives me YAML Invalid error, when just copy pasting a script from forum or anyplace.
Also, can you guys point me to some resource where some of these basic things clarified?

**Comment by Zachatoo** [comment]
1. https://silentvoid13.github.io/Templater/commands/execution-command.html#how-to-output-a-value-from-a-javascript-execution-command-
2. Can you provide a specific example?. I have not had that issue before, and I use Templater in frontmatter extensively.
3. The docs are great https://silentvoid13.github.io/Templater/introduction.html
